tests/robot-tests/tests/general_public/miscellaneous.robot

```diff
@@ -60,7 +63,8 @@ Validate Cookies page
 
 Disable google analytics
     [Tags]  HappyPath   NotAgainstLocal
-    user clicks element   id:cookieSettingsForm-googleAnalytics-off
+    # NOTE: think there is a problem with the frontend. You'd think cookieSettingsForm would only be in the id once...
```

Comment:
I think you can fix this as part of your PR - its line 129 in CookiesPage.tsx - I think it should just be `id="googleAnalytics"` - it seems the form automatically suffixes the id from the Form component.